### ISSUE TITLE
Pango unknown glyph with certain fonts

### ISSUE THREAD

**Issue by NBonaparte** [geommer/yabar#127]
Pango has a certain "feature" where spaces use the font for the character before the space. As a result, there can be some undesirable results:
* fonts without space characters will display tofu symbols ([example here](https://github.com/eosrei/emojione-color-font/issues/2)).
* when switching between symbolic and monospace fonts, the width will be incorrect (it uses the symbol font's space).

One solution to the former is to add a fallback attribute to each space. With the latter, a specific font family can be applied through another attribute.

So far, the only workaround I've come up with is looping through each buffer text, finding any spaces, and applying a fallback attribute to every space.

**Comment by jacksgt** [comment]
It is already possible to specify multiple (fallback) fonts in yabar, e.g.:

    font: "Droid Sans, FontAwesome Bold 9";

Doesn't this cover the missing spaces?

**Comment by NBonaparte** [comment]
No, for some reason it doesn't switch fonts for spaces, so we have to do it manually.

**Comment by soyuka** [comment]
FontAwesome v5 apparently has no more space character as I'm facing this issue :|.

**Comment by gh67uyyghj** [comment]
I use Archlinux and something happened after my last update a week ago. Some icons aren't showing correctly. But I don't my problem is related to this issue. Something happened with FontAwesome 5 I guess

**Comment by Narrat** [comment]
Regarding the FontAwesome5 thing, there is this Arch bug report: https://bugs.archlinux.org/task/56899

**Comment by NBonaparte** [comment]
Can't believe Pango still hasn't fixed that. Does the `fix_pango` branch still fixes this issue? I just rebased it onto master.

**Comment by Narrat** [comment]
With the fix_pango branch and following fonts `font: "DejaVu Sans Mono 10, Siji 10, MaterialDesignIcons 10";` I'm still having those unknown glyphs

**Comment by soyuka** [comment]
Tried the `fix_pango` branch without success. I'm using Source Code Pro and awesome 5.

**Comment by soyuka** [comment]
Updating/installing these packages fixed the issue for me:

`otf-font-awesome ttf-font-awesome`
